src/dragonball/src/api/v1/vmm_action.rs

```diff
@@ -813,6 +849,99 @@ impl VmmService {
             .map_err(VmmActionError::FsDevice)
     }
 
+    #[cfg(feature = "host-device")]
+    fn add_vfio_device(&self, vmm: &mut Vmm, config: HostDeviceConfig) -> VmmRequestResult {
+        let vm = vmm.get_vm_mut().ok_or(VmmActionError::HostDeviceConfig(
+            VfioDeviceError::InvalidVMID,
```

Comment:
Nit: `InvalidVMID` -> `InvalidVmId`

Reply:
I'll raise a separate PR to fix this.

Reply:
https://github.com/kata-containers/kata-containers/issues/8747 is raised for this problem.